tests/dialects/test_vector.py

```diff
@@ -441,7 +441,7 @@ def test_vector_masked_load_verify_result_vector_rank():
 
     with pytest.raises(Exception) as exc_info:
         maskedload.verify()
-    assert exc_info.value.args[0] == "Expected a rank 1 result vector."
+    assert "Expected vector rank to be 1, got 2." in exc_info.value.args[0]
```

Comment:
Why is it `in` now and not `==` anymore? Is this because of nested exceptions?

Reply:
Yes, the error which I get atm is this 
```
xdsl.utils.exceptions.DiagnosticException: vector.maskedstore operation does not verify

vector.maskedstore(%<UNKNOWN> : !memref<[1 : !index], !i32>, %<UNKNOWN> : !vector<[2 : !index], !i32>, %<UNKNOWN> : !vector<[1 : !index], !i32>)
^^^^^^^^^^^^^^^^^^-------------------------
| operand at position 2 does not verify!
| Expected vector type to be !i1, got !i32.
-------------------------------------------
-------------------^^^^^^^^^^--------------------------------------------------------------------
| ERROR: SSAValue is not part of the IR, are you sure all operations are added before their uses?
-------------------------------------------------------------------------------------------------
-------------------------------------------------------------^^^^^^^^^^--------------------------
| ERROR: SSAValue is not part of the IR, are you sure all operations are added before their uses?
-------------------------------------------------------------------------------------------------
-------------------------------------------------------------------------------------------------------^^^^^^^^^^
| ERROR: SSAValue is not part of the IR, are you sure all operations are added before their uses?
-----------------------------------------------------------------------------------------------------------------
```

Reply:
Ah okay, during printing, everything gets weird. I guess this is one of the cases where we should not print the SSAValue error. That's just confusing overhead, that is not useful to anyone...